TitanRobotics2024/src/main/java/frc/robot/Subsystem/ModifiedMotors.java

```diff
@@ -1,39 +1,51 @@
 package frc.robot.Subsystem;
 
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
 import edu.wpi.first.wpilibj.motorcontrol.MotorController;
 import edu.wpi.first.wpilibj.motorcontrol.PWMVictorSPX;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
 public class ModifiedMotors implements Subsystem {
 
-
-    @Override
-    public void start() {
-        // TODO Auto-generated method stub
-        throw new UnsupportedOperationException("Unimplemented method 'start'");
-    }
+    // @Override
+    // public void start() {
+    // // TODO Auto-generated method stub
+    // throw new UnsupportedOperationException("Unimplemented method 'start'");
+    // }
 
     @Override
     public void update() {
         // TODO Auto-generated method stub
         throw new UnsupportedOperationException("Unimplemented method 'update'");
     }
-    
-     private int portNumber;
+
+    private int portNumber;
     private final MotorController motor;
-    
-    public ModifiedMotors(int portNumber) 
```

Comment:
Maybe encapsulate the motor initialization logic into separate methods for improved readability and maintainability
``` 
public ModifiedMotors(int portNumber, String motorType) {
        this.portNumber = portNumber;
        MotorController motorTemporarily = null;
        switch (motorType) {
            case "PWMVictorSPX":
                motorTemporarily = initializePWMVictorSPX(portNumber);
                break;
            case "CANVictorSPX":
                motorTemporarily = initializeCANVictorSPX(portNumber);
                break;
            default:
                System.err.println("Error: motors not activated");
        }
        motor = motorTemporarily;
    }

    private MotorController initializePWMVictorSPX(int portNumber) {
        try {
            return new PWMVictorSPX(portNumber);
        } catch (Exception e) {
            System.err.println("Error: Port Not Activated " + portNumber);
            return null;
        }
    }

    private MotorController initializeCANVictorSPX(int portNumber) {
        try {
            return new WPI_VictorSPX(portNumber);
        } catch (Exception e) {
            System.err.println("Error: CANID Not Activated " + portNumber);
            return null;
        }
    }
```